### PR TITLE
User login with validation with a database.txt file. 

### PR DESCRIPTION
**Pull request by o3dwade**
Will need work on html file but on a different iteration.

### CODE REVIEW

**Comment by XanaduBarchetta** [comment]
requirements.txt has some packages in it which are not used in the code.

File: spades.py [hunk 1]
```diff
@@ -1,8 +1,111 @@
-from flask import Flask
+import bcrypt
+from getpass import getpass
+from pathlib import Path
```
Comment (XanaduBarchetta) [inline spades.py:3]: Doesn't look like these are used.

File: spades.py [hunk 2]
```diff
@@ -1,8 +1,111 @@
...
+
+def __verifyExistingUser(username):
+    users = __getUsers()
+    return username not in users
+
+def __registerUser(username, password):
+    hashedPassword = sha256_crypt.encrypt(password)
+
+    # TODO Point to database instead of a text file
+    with open("database.txt", "a") as file:
```
Comment (XanaduBarchetta) [inline spades.py:100]: This fails if the file doesn't exist. I think in Python you have to use `"a+"` to create the file if it's missing.

Comment (XanaduBarchetta) [reply spades.py:100]: Of course this isn't really an issue since we won't be using a text file.

**Review by XanaduBarchetta — APPROVED** [review]

File: spades.py (hunk 2 above)
Comment (o3dwade) [reply spades.py:100]: I tested it, it did not fail. Also, if there is not 'database.txt' file it's the right approach to fail the application completely as that is a huge security concern.

**Pull request merged**